src/main.rs

```diff
@@ -306,13 +300,17 @@ fn main() {
                 .short('p')
                 .long("process")
                 .value_name("NAME")
-                .help("Name of running process to check")
-                .takes_value(true),
+                .help("Name of running process to check"),
         )
+        .arg(Arg::new("pid").long("pid").value_name("PID").help(
```

Comment:
Please align to existing arg formatting with `.long`, `.value_name` and `.help` on separate lines.